Design.md

```diff
@@ -55,9 +55,17 @@ Users will be encouraged to run most of their production systems on `stable`, an
 
 ### Development Refs
 
-There will also be some additional unversioned refs for the convenience of Fedora CoreOS developers.  These will be public, but won't be exposed to users in the same way as production refs: they might be in a different repo, or in the same repo but not listed in the summary file.  None of these are contractual; they might go away if we don't find them useful.
+Development for the next `testing` and `next` releases will occur in development refs.  These refs will be public, but will be stored in a different ostree repo from production refs.
```

Comment:
Minor: I tend to use "OSTree" when not referring to the command-line tool itself.